k_p_x:   18000.0    
k_i_x:   0.0     
k_d_x:   9000.0    
k_a_x:   0.0     

k_p_y:   15000.0    
k_i_y:   0.0     
k_d_y:   6000.0    
k_a_y:   0.0 

k_p_z:    100.0
k_i_z:    4.0
k_d_z:    60.0
k_a_z:    0.0

k_p_yaw:  20.0
k_i_yaw:  1.0
k_d_yaw:  5.0
k_a_yaw:  0.0
